frontend/src/modules/scaffold/views/scaffoldUsersView.js

```diff
@@ -0,0 +1,98 @@
+define([ 'core/origin', 'backbone-forms' ], function(Origin, BackboneForms) {
+  var ScaffoldUsersView = Backbone.Form.editors.Base.extend({
+    tagName: 'input',
+    className: 'scaffold-users',
+    idField: 'email',
+    events: {
+      'change': function() { this.trigger('change', this); },
+      'focus': function() { this.trigger('focus', this); },
+      'blur': function() { this.trigger('blur', this); }
+    },
+
+    render: function() {
+      this.fetchUsers(this.initSelectize);
+      return this;
+    },
+
+    renderItem: function(item, escape) {
+      var name = item.firstName && item.lastName ? escape(item.firstName  + ' ' +  item.lastName) : '';
+      var email = escape(item.email);
+
+      var $div = $('<div class="option">')
+        .append('<span class="name">' + (name || email) + '</span>');
+
+      if(name) {
+        $div.append('<span class="email">&lt;' + email + '&gt;</span>');
+      }
+      if(item.disabled) {
+        $div.attr('data-disabled', true);
+      }
```

Comment:
Could switch to object-style syntax if you fancied:
```js
var $div = $('<div>', { 'class': 'option', 'data-disabled': item.disabled })
  .append($('<span>', { 'class': 'name', text: name || email }));

if (name) {
  $div.append($('<span>', { 'class': 'email', text: '<' + email + '>' }));
}

return $div;
```

Reply:
Have replaced the above with a hbs template